shared-libs/environment/test/index.spec.js

```diff
@@ -0,0 +1,73 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+const rewire = require('rewire');
+
+let originalEnv;
+const stubProcessEnv = (props) => {
+  originalEnv = { ...process.env };
+  process.env = { ...process.env, ...props };
+};
+
+describe('environment', () => {
+  afterEach(() => {
+    sinon.restore();
+    process.env = originalEnv;
+  });
+
+  it('should exit if no couchurl', async () => {
+    sinon.stub(process, 'exit');
+    stubProcessEnv({ COUCH_URL: '' });
+    rewire('../src/index');
+    expect(process.exit.calledWith(1)).to.be.true;
```

Comment:
Minor, but we should also be able to also stub and assert the logger in these first two tests, right?